utils/api.js

```
@@ -67,4 +72,4 @@ const acceptTerms = ({ token }) => {
   });
 }
 
-export { login, signUp, validateUserToken, recoverPassword, resetPassword, acceptTerms };
+export { login, googleAuth, signUp, validateUserToken, recoverPassword, resetPassword, acceptTerms };
```

ESLint

[error] Replace `·login,·googleAuth,·signUp,·validateUserToken,·recoverPassword,·resetPassword,·acceptTerms·` with `⏎··login,⏎··googleAuth,⏎··signUp,⏎··validateUserToken,⏎··recoverPassword,⏎··resetPassword,⏎··acceptTerms,⏎`

(prettier/prettier)